src/llmq/quorums_dkgsessionmgr.cpp

```diff
@@ -24,27 +24,29 @@ CDKGSessionManager::CDKGSessionManager(CDBWrapper& _llmqDb, CBLSWorker& _blsWork
     llmqDb(_llmqDb),
     blsWorker(_blsWorker)
 {
+    for (const auto& qt : Params().GetConsensus().llmqs) {
```

Comment:
why's this called qt? maybe `quorum` or `quorumThread` would be clearer (I'm not actually sure what exactly this variable represents)

Reply:
I have no idea why its called like that i just moved it around a bit. But it represents a key/value pair from the `Consensus::Params::llmqs` map which cointains `LLMQType, LLMQParams` pairs for the available LLMQs.

Reply:
What if we make it not auto so it's clearer what this is referring to. IMO we should only really be using auto where by the name of it and the surrounding code it's obvious. I guess it's obvious in the sense you can look at `llmqs` and find what that is...

Reply:
>  guess it's obvious in the sense you can look at `llmqs` and find what 

Hm.. so for me that `auto` style isn't an issue at all because the declaration of `llmqs` is only a click away. But thats obviously a personal preference thing and at the end anyway kind of unrelated to this PR though because it's used like this all over the place :D

Reply:
Yeah, it's fine how it is. Just makes code review easier when it's obvious what variables mean